requirements.txt

```diff
@@ -1,9 +1,10 @@
 bitarray==1.0.1
 cffi>=1.7
+clkhash>=0.11
+Cython==0.29.10
+hypothesis==4.33.0
 pytest>=3.4
 pytest-cov>=2.5
-clkhash==0.13.0
+bitarray-hardbyte==1.1.0
```

Comment:
It's a preference, but would be good to keep the requirements in alphabetical order.